hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -984,7 +984,7 @@ func isArchAndPlatformSupported(nodePool *hyperv1.NodePool) bool {
 			supported = true
 		}
 	case hyperv1.AgentPlatform:
```

Comment:
I would add also the `hyperv1.NonePlatform` as another case, even not being supported, I think it could be useful for dev envs.